templates/dapp-boilerplate/.gitignore

```diff
@@ -0,0 +1,5 @@
+package-lock.json
```

Comment:
All this frontend stuff should probably not be here since we've got a gitignore in the frontend directory.

Reply:
all these stuff are files on the root folder